docs/product-manuals/cloud-console/manage-clusters/manage-api-clients.md

```diff
@@ -3,37 +3,44 @@ id: manage-api-clients
+:::note
+Access tokens have a validity period that can be found in the access token. After this time, a new access token must be requested.
+:::
 
-- Access tokens have a validity period that can be found in the access token. After this time a new access token must be requested.
-- The auth service has a built-in rate limiting. If too many token requests are executed in a short time, the client is blocked for a certain time. Since the access tokens have a certain validity period, they must be cached on the client side.
+:::note
+The auth service has a built-in rate limit. If too many token requests are executed in a short time, the client is blocked for a certain time. Since the access tokens have a certain validity period, they must be cached on the client side.
 
-The officially offered client libraries (as well as the node client) have already integrated the auth routine and take care of obtaining and refreshing an access token and make use of a local cache.
+The officially offered client libraries (as well as the node client) have already integrated with the auth routine, handle obtaining and refreshing an access token, and make use of a local cache.
+:::
 
 ### Create a client
 
-To create a new client you have to navigate into the `API` tab:
+To create a client, take the following steps:
+
+1. Navigate into the `API` tab.
 
 ![cluster-details](../../../guides/getting-started/img/cluster-detail-clients.png)
 
-Click **Create New Client** to create a new client and name your client accordingly.
+2. Click **Create New Client** to create a new client and name your client accordingly.
 
 ![create-client](../../../guides/getting-started/img/cluster-details-create-client.png)
 
-Make sure you keep the generated client credentials in a safe place - the **client secret** will not be shown again. For your convenience, you can also download the client information to your computer.
+3. Ensure you keep the generated client credentials in a safe place. The **client secret** will not be shown again. For your convenience, you can also download the client information to your computer.
 
 ![created-client](../../../guides/getting-started/img/cluster-details-created-client.png)
 
-The downloaded file contains all necessary information to later on communicate with your zeebe instance:
+The downloaded file contains all necessary information to communicate with your Zeebe instance in the future:
+
+- `ZEEBE_ADDRESS`: Address where your cluster can be reached.
+- `ZEEBE_CLIENT_ID` and `ZEEBE_CLIENT_SECRET`: Credentials to request a new access token.
+- `ZEEBE_AUTHORIZATION_SERVER_URL`: A new token can be requested at this address, using the
 
-- `ZEEBE_ADDRESS`: address where your cluster can be reached.
-- `ZEEBE_CLIENT_ID` and `ZEEBE_CLIENT_SECRET`: credentials to request a new access token.
-- `ZEEBE_AUTHORIZATION_SERVER_URL`: a new token can be requested at this address, using the
+[//]:# (Looks like this doc cuts off. How should we finish the sentence?)
```

Comment:
Oops ;) We can just cut off before the comma: `A new token can be requested at this address.`